__tests__/app.test.js

```diff
@@ -42,3 +43,29 @@ describe("GET /api/topics", () => {
     })
   })
 })
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with a single article object to the client", () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then((response) => {
+      expect(typeof response.body.article.article_id).toBe('number');
+      expect(typeof response.body.article.author).toBe('string');
+      expect(typeof response.body.article.title).toBe('string');
+      expect(typeof response.body.article.body).toBe('string');
+      expect(typeof response.body.article.topic).toBe('string');
+      expect(typeof response.body.article.created_at).toBe('string');
+      expect(typeof response.body.article.votes).toBe('number');
+      expect(typeof response.body.article.article_img_url).toBe('string');
```

Comment:
This is not a bad way of doing asserting these tests, but since you are actually getting only one result, you know exactly what it the values of this article is, at the very least it you want to check that the "article_id" is the same as the ID that you requested, in this case, ".toBe(1)".